docs/sp-migration/sp-migration.md

```diff
@@ -102,17 +110,18 @@ kubectl get managed
 kubectl get provider.pkg
 ```
 
-10. Find new dependencies and add them to `dependsOn` field in the `crossplane.yaml` file:
+10. If you want to add smaller providers to the configuration's `dependsOn` list, please follow the guide and build/push/update
```

Comment:
What is the referred guide here? We may consider giving a link.

Reply:
Forgot to add parentheses, thanks for catching :)